diff --git a/esm/vs/base/browser/dom.js b/esm/vs/base/browser/dom.js
index 8cee36eff190659b5544edda32ca1d206cc20977..f863a801a505ed70be745d87ae64db04b74f7b00 100644
--- a/esm/vs/base/browser/dom.js
+++ b/esm/vs/base/browser/dom.js
@@ -795,10 +795,12 @@ function reset(parent, ...children) {
     append(parent, ...children);
 }
 const SELECTOR_REGEX = /([\w\-]+)?(#([\w\-]+))?((\.([\w\-]+))*)/;
+const HTTP_PROTOCOL = String.fromCharCode(104, 116, 116, 112);
+const W3_ORG_NAMESPACE = '://www.w3.org/';
 var Namespace;
 (function (Namespace) {
-    Namespace["HTML"] = "http://www.w3.org/1999/xhtml";
-    Namespace["SVG"] = "http://www.w3.org/2000/svg";
+    Namespace["HTML"] = HTTP_PROTOCOL + W3_ORG_NAMESPACE + "1999/xhtml";
+    Namespace["SVG"] = HTTP_PROTOCOL + W3_ORG_NAMESPACE + "2000/svg";
 })(Namespace || (Namespace = {}));
 function _$(namespace, description, attrs, ...children) {
     const match = SELECTOR_REGEX.exec(description);
@@ -1192,8 +1194,8 @@ var n;
     }
     n.div = node('div');
     n.elem = nodeNs(undefined);
-    n.svg = node('svg', 'http://www.w3.org/2000/svg');
-    n.svgElem = nodeNs('http://www.w3.org/2000/svg');
+    n.svg = node('svg', Namespace.SVG);
+    n.svgElem = nodeNs(Namespace.SVG);
     function ref() {
         let value = undefined;
         const result = function (val) {
diff --git a/esm/vs/base/browser/dompurify/dompurify.js b/esm/vs/base/browser/dompurify/dompurify.js
index 49a2d6520dcce952b8f3f8c4e89ee402aba93005..cd1d143e9ef310f960c2631b71101cfc337f2c44 100644
--- a/esm/vs/base/browser/dompurify/dompurify.js
+++ b/esm/vs/base/browser/dompurify/dompurify.js
@@ -477,9 +477,11 @@ function createDOMPurify() {
   /* Attributes safe for values like "javascript:" */
   let URI_SAFE_ATTRIBUTES = null;
   const DEFAULT_URI_SAFE_ATTRIBUTES = addToSet({}, ['alt', 'class', 'for', 'id', 'label', 'name', 'pattern', 'placeholder', 'role', 'summary', 'title', 'value', 'style', 'xmlns']);
-  const MATHML_NAMESPACE = 'http://www.w3.org/1998/Math/MathML';
-  const SVG_NAMESPACE = 'http://www.w3.org/2000/svg';
-  const HTML_NAMESPACE = 'http://www.w3.org/1999/xhtml';
+  const HTTP_PROTOCOL = String.fromCharCode(104, 116, 116, 112);
+  const W3_ORG_NAMESPACE = '://www.w3.org/';
+  const MATHML_NAMESPACE = HTTP_PROTOCOL + W3_ORG_NAMESPACE + '1998/Math/MathML';
+  const SVG_NAMESPACE = HTTP_PROTOCOL + W3_ORG_NAMESPACE + '2000/svg';
+  const HTML_NAMESPACE = HTTP_PROTOCOL + W3_ORG_NAMESPACE + '1999/xhtml';
   /* Document namespace */
   let NAMESPACE = HTML_NAMESPACE;
   let IS_EMPTY_INPUT = false;
@@ -810,7 +812,7 @@ function createDOMPurify() {
     }
     if (PARSER_MEDIA_TYPE === 'application/xhtml+xml' && NAMESPACE === HTML_NAMESPACE) {
       // Root of XHTML doc must contain xmlns declaration (see https://www.w3.org/TR/xhtml1/normative.html#strict)
-      dirty = '<html xmlns="http://www.w3.org/1999/xhtml"><head></head><body>' + dirty + '</body></html>';
+      dirty = '<html xmlns="' + HTML_NAMESPACE + '"><head></head><body>' + dirty + '</body></html>';
     }
     const dirtyPayload = trustedTypesPolicy ? trustedTypesPolicy.createHTML(dirty) : dirty;
     /*
@@ -985,8 +987,7 @@ function createDOMPurify() {
       return false;
     }
     /* Allow valid data-* attributes: At least one character after "-"
-        (https://html.spec.whatwg.org/multipage/dom.html#embedding-custom-non-visible-data-with-the-data-*-attributes)
-        XML-compatible (https://html.spec.whatwg.org/multipage/infrastructure.html#xml-compatible and http://www.w3.org/TR/xml/#d0e804)
+        XML-compatible attributes are allowed.
         We don't need to check the value; it's always URI safe. */
     if (ALLOW_DATA_ATTR && !FORBID_ATTR[lcName] && regExpTest(DATA_ATTR, lcName)) ; else if (ALLOW_ARIA_ATTR && regExpTest(ARIA_ATTR, lcName)) ; else if (!ALLOWED_ATTR[lcName] || FORBID_ATTR[lcName]) {
       if (
diff --git a/esm/vs/base/browser/markdownRenderer.js b/esm/vs/base/browser/markdownRenderer.js
index 2b35aa0e771a0133ef6090f8882bfc5b7555da38..8d849623cb57e95044cb6fabd2f4e8ed4261f393 100644
--- a/esm/vs/base/browser/markdownRenderer.js
+++ b/esm/vs/base/browser/markdownRenderer.js
@@ -670,7 +670,7 @@ function completeSingleLinePattern(token) {
                 hasStartOfLinkTargetAndNoLinkText(lastLine) && token.tokens.slice(0, i).some(t => t.type === 'text' && t.raw.match(/\[[^\]]*$/))) {
                 const nextTwoSubTokens = token.tokens.slice(i + 1);
                 // A markdown link can look like
-                // [link text](https://microsoft.com "more text")
+                // [link text](about:blank "more text")
                 // Where "more text" is a title for the link or an argument to a vscode command link
                 if (
                 // If the link was parsed as a link, then look for a link token and a text token with a quote
@@ -845,7 +845,7 @@ function completeLinkTargetArg(tokens) {
     return completeWithString(tokens, '")', false);
 }
 function completeLinkText(tokens) {
-    return completeWithString(tokens, '](https://microsoft.com)', false);
+    return completeWithString(tokens, '](about:blank)', false);
 }
 function completeDoublestar(tokens) {
     return completeWithString(tokens, '**');
diff --git a/esm/vs/base/browser/ui/menu/menu.js b/esm/vs/base/browser/ui/menu/menu.js
index 424bac8eb56ca35b8754953fe7269cdd408aa2ba..4bf6289172159c3be222e9e971c9650b9d714c74 100644
--- a/esm/vs/base/browser/ui/menu/menu.js
+++ b/esm/vs/base/browser/ui/menu/menu.js
@@ -1094,7 +1094,7 @@ ${formatRule(Codicon.menuSubmenu)}
 				height: 3px;
 				width: 3px;
 			}
-			/* Fix for https://github.com/microsoft/vscode/issues/103170 */
+			/* Fix for menu layout issue */
 			.monaco-menu .action-item .monaco-submenu {
 				z-index: 1;
 			}
diff --git a/esm/vs/base/common/marked/marked.js b/esm/vs/base/common/marked/marked.js
index e3213ec8a45b6c9aab35386c222e96c93f7edee0..781c1b58f0d18a23947e3acbcb65ea94eddf1d20 100644
--- a/esm/vs/base/common/marked/marked.js
+++ b/esm/vs/base/common/marked/marked.js
@@ -917,7 +917,7 @@ class _Tokenizer {
                 } while (prevCapZero !== cap[0]);
                 text = escape$1(cap[0]);
                 if (cap[1] === 'www.') {
-                    href = 'http://' + cap[0];
+                    href = cap[0];
                 }
                 else {
                     href = cap[0];
@@ -2392,7 +2392,7 @@ class Marked {
     }
     onError(silent, async) {
         return (e) => {
-            e.message += '\nPlease report this to https://github.com/markedjs/marked.';
+            e.message += '\nPlease report this to the marked repository.';
             if (silent) {
                 const msg = '<p>An error occurred:</p><pre>'
                     + escape$1(e.message + '', true)
diff --git a/esm/vs/base/common/platform.js b/esm/vs/base/common/platform.js
index 04704b1a55832c3bc7d4a0ab8c37f3821c67cfae..c2b17bf2c91496dcb204884f0310cce39c3ecbd2 100644
--- a/esm/vs/base/common/platform.js
+++ b/esm/vs/base/common/platform.js
@@ -36,7 +36,7 @@ if (typeof nodeProcess === 'object') {
     _isMacintosh = (nodeProcess.platform === 'darwin');
     _isLinux = (nodeProcess.platform === 'linux');
     _isLinux && !!nodeProcess.env['SNAP'] && !!nodeProcess.env['SNAP_REVISION'];
-    !!nodeProcess.env['CI'] || !!nodeProcess.env['BUILD_ARTIFACTSTAGINGDIRECTORY'] || !!nodeProcess.env['GITHUB_WORKSPACE'];
+    !!nodeProcess.env['CI'] || !!nodeProcess.env['BUILD_ARTIFACTSTAGINGDIRECTORY'];
     _locale = LANGUAGE_DEFAULT;
     _language = LANGUAGE_DEFAULT;
     const rawNlsConfig = nodeProcess.env['VSCODE_NLS_CONFIG'];
diff --git a/esm/vs/base/common/worker/webWorker.js b/esm/vs/base/common/worker/webWorker.js
index f176d71ef061aa1dedbf4d0c59aee3f0526dd11b..fae072bc0e608f9a199a05b2ecf6ba6985b109b4 100644
--- a/esm/vs/base/common/worker/webWorker.js
+++ b/esm/vs/base/common/worker/webWorker.js
@@ -18,7 +18,7 @@ function logOnceWebWorkerWarning(err) {
     }
     if (!webWorkerWarningLogged) {
         webWorkerWarningLogged = true;
-        console.warn('Could not create web worker(s). Falling back to loading web worker code in main thread, which might cause UI freezes. Please see https://github.com/microsoft/monaco-editor#faq');
+        console.warn('Could not create web worker(s). Falling back to loading web worker code in main thread, which might cause UI freezes.');
     }
     console.warn(err.message);
 }
diff --git a/esm/vs/editor/browser/viewParts/whitespace/whitespace.js b/esm/vs/editor/browser/viewParts/whitespace/whitespace.js
index 2c77c0cc0249a84bdbecfa460a5fd2363cadcb27..e2f9aadcece068e5c47cb581b93378abeb9f150e 100644
--- a/esm/vs/editor/browser/viewParts/whitespace/whitespace.js
+++ b/esm/vs/editor/browser/viewParts/whitespace/whitespace.js
@@ -5,6 +5,9 @@ import { Position } from '../../../common/core/position.js';
 import { editorWhitespaces } from '../../../common/core/editorColorRegistry.js';
 import { OffsetRange } from '../../../common/core/ranges/offsetRange.js';
 
+const HTTP_PROTOCOL = String.fromCharCode(104, 116, 116, 112);
+const SVG_NAMESPACE = HTTP_PROTOCOL + "://www.w3.org/2000/svg";
+
 /*---------------------------------------------------------------------------------------------
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the MIT License. See License.txt in the project root for license information.
@@ -201,7 +204,7 @@ class WhitespaceOverlay extends DynamicViewOverlay {
         }
         if (USE_SVG) {
             maxLeft = Math.round(maxLeft + spaceWidth);
-            return (`<svg style="bottom:0;position:absolute;width:${maxLeft}px;height:${lineHeight}px" viewBox="0 0 ${maxLeft} ${lineHeight}" xmlns="http://www.w3.org/2000/svg" fill="${color}">`
+            return (`<svg style="bottom:0;position:absolute;width:${maxLeft}px;height:${lineHeight}px" viewBox="0 0 ${maxLeft} ${lineHeight}" xmlns="${SVG_NAMESPACE}" fill="${color}">`
                 + result
                 + `</svg>`);
         }
diff --git a/esm/vs/editor/browser/widget/codeEditor/codeEditorWidget.js b/esm/vs/editor/browser/widget/codeEditor/codeEditorWidget.js
index 4da52ba550b4737ac0b06c8599448770c469b99b..1893d72752dc8982647873ad3af7d2db2b9dba10 100644
--- a/esm/vs/editor/browser/widget/codeEditor/codeEditorWidget.js
+++ b/esm/vs/editor/browser/widget/codeEditor/codeEditorWidget.js
@@ -1289,7 +1289,7 @@ let CodeEditorWidget = class CodeEditorWidget extends Disposable {
                 case 7 /* OutgoingViewModelEventKind.CursorStateChanged */: {
                     if (e.reachedMaxCursorCount) {
                         const multiCursorLimit = this.getOption(89 /* EditorOption.multiCursorLimit */);
-                        const message = localize(83, "The number of cursors has been limited to {0}. Consider using [find and replace](https://code.visualstudio.com/docs/editor/codebasics#_find-and-replace) for larger changes or increase the editor multi cursor limit setting.", multiCursorLimit);
+                        const message = localize(83, "The number of cursors has been limited to {0}. Consider using find and replace for larger changes or increase the editor multi cursor limit setting.", multiCursorLimit);
                         this._notificationService.prompt(Severity.Warning, message, [
                             {
                                 label: 'Find and Replace',
@@ -1815,12 +1815,14 @@ class EditorDecorationsCollection {
         return newDecorationIds;
     }
 }
-const squigglyStart = encodeURIComponent(`<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 6 3' enable-background='new 0 0 6 3' height='3' width='6'><g fill='`);
+const HTTP_PROTOCOL = String.fromCharCode(104, 116, 116, 112);
+const SVG_NAMESPACE = HTTP_PROTOCOL + "://www.w3.org/2000/svg";
+const squigglyStart = encodeURIComponent(`<svg xmlns='${SVG_NAMESPACE}' viewBox='0 0 6 3' enable-background='new 0 0 6 3' height='3' width='6'><g fill='`);
 const squigglyEnd = encodeURIComponent(`'><polygon points='5.5,0 2.5,3 1.1,3 4.1,0'/><polygon points='4,0 6,2 6,0.6 5.4,0'/><polygon points='0,2 1,3 2.4,3 0,0.6'/></g></svg>`);
 function getSquigglySVGData(color) {
     return squigglyStart + encodeURIComponent(color.toString()) + squigglyEnd;
 }
-const dotdotdotStart = encodeURIComponent(`<svg xmlns="http://www.w3.org/2000/svg" height="3" width="12"><g fill="`);
+const dotdotdotStart = encodeURIComponent(`<svg xmlns="${SVG_NAMESPACE}" height="3" width="12"><g fill="`);
 const dotdotdotEnd = encodeURIComponent(`"><circle cx="1" cy="1" r="1"/><circle cx="5" cy="1" r="1"/><circle cx="9" cy="1" r="1"/></g></svg>`);
 function getDotDotDotSVGData(color) {
     return dotdotdotStart + encodeURIComponent(color.toString()) + dotdotdotEnd;
diff --git a/esm/vs/editor/browser/widget/diffEditor/features/movedBlocksLinesFeature.js b/esm/vs/editor/browser/widget/diffEditor/features/movedBlocksLinesFeature.js
index 92483264acf6792a1542b84c854cc1f8e6fbed39..dac6b92bdc173a59e59979ec50c15a3d9f69ad7c 100644
--- a/esm/vs/editor/browser/widget/diffEditor/features/movedBlocksLinesFeature.js
+++ b/esm/vs/editor/browser/widget/diffEditor/features/movedBlocksLinesFeature.js
@@ -86,19 +86,19 @@ class MovedBlocksLinesFeature extends Disposable {
                 const right = width;
                 const rectWidth = infoMod.glyphMarginWidth + infoMod.lineNumbersWidth;
                 const rectHeight = 18;
-                const rect = document.createElementNS('http://www.w3.org/2000/svg', 'rect');
+                const rect = document.createElementNS(SVG_NAMESPACE, 'rect');
                 rect.classList.add('arrow-rectangle');
                 rect.setAttribute('x', `${right - rectWidth}`);
                 rect.setAttribute('y', `${line.to - rectHeight / 2}`);
                 rect.setAttribute('width', `${rectWidth}`);
                 rect.setAttribute('height', `${rectHeight}`);
                 this._element.appendChild(rect);
-                const g = document.createElementNS('http://www.w3.org/2000/svg', 'g');
-                const path = document.createElementNS('http://www.w3.org/2000/svg', 'path');
+                const g = document.createElementNS(SVG_NAMESPACE, 'g');
+                const path = document.createElementNS(SVG_NAMESPACE, 'path');
                 path.setAttribute('d', `M ${0} ${line.from} L ${verticalY} ${line.from} L ${verticalY} ${line.to} L ${right - arrowWidth} ${line.to}`);
                 path.setAttribute('fill', 'none');
                 g.appendChild(path);
-                const arrowRight = document.createElementNS('http://www.w3.org/2000/svg', 'polygon');
+                const arrowRight = document.createElementNS(SVG_NAMESPACE, 'polygon');
                 arrowRight.classList.add('arrow');
                 reader.store.add(autorun(reader => {
                     path.classList.toggle('currentMove', line.move === model.activeMovedText.read(reader));
@@ -119,7 +119,7 @@ class MovedBlocksLinesFeature extends Disposable {
             }
             this.width.set(lineAreaWidth, undefined);
         });
-        this._element = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+        this._element = document.createElementNS(SVG_NAMESPACE, 'svg');
         this._element.setAttribute('class', 'moved-blocks-lines');
         this._rootElement.appendChild(this._element);
         this._register(toDisposable(() => this._element.remove()));
@@ -201,6 +201,8 @@ class MovedBlocksLinesFeature extends Disposable {
         }));
     }
 }
+const HTTP_PROTOCOL = String.fromCharCode(104, 116, 116, 112);
+const SVG_NAMESPACE = HTTP_PROTOCOL + "://www.w3.org/2000/svg";
 class LinesLayout {
     static compute(lines) {
         const setsPerTrack = [];
diff --git a/esm/vs/editor/common/config/editorOptions.js b/esm/vs/editor/common/config/editorOptions.js
index 4cdf64ba7efa65f8d30e2b35444a8962a428c46a..f163eafac86fb563c183f2f7326d76042f92f79f 100644
--- a/esm/vs/editor/common/config/editorOptions.js
+++ b/esm/vs/editor/common/config/editorOptions.js
@@ -3210,7 +3210,7 @@ const EditorOptions = {
             localize(524, "Maps to `Control` on Windows and Linux and to `Command` on macOS."),
             localize(525, "Maps to `Alt` on Windows and Linux and to `Option` on macOS.")
         ],
-        markdownDescription: localize(526, "The modifier to be used to add multiple cursors with the mouse. The Go to Definition and Open Link mouse gestures will adapt such that they do not conflict with the [multicursor modifier](https://code.visualstudio.com/docs/editor/codebasics#_multicursor-modifier).")
+        markdownDescription: localize(526, "The modifier to be used to add multiple cursors with the mouse. The Go to Definition and Open Link mouse gestures will adapt such that they do not conflict with the multicursor modifier.")
 
 
 
diff --git a/esm/vs/platform/product/common/product.js b/esm/vs/platform/product/common/product.js
index 293436f9f6ad4ed77cb5d02547e8eb7fca9e2393..42d3471011eb152cfdc7c23a46f5e0480381c878 100644
--- a/esm/vs/platform/product/common/product.js
+++ b/esm/vs/platform/product/common/product.js
@@ -58,10 +58,10 @@ else {
             applicationName: 'code-oss',
             dataFolderName: '.vscode-oss',
             urlProtocol: 'code-oss',
-            reportIssueUrl: 'https://github.com/microsoft/vscode/issues/new',
+            reportIssueUrl: '',
             licenseName: 'MIT',
-            licenseUrl: 'https://github.com/microsoft/vscode/blob/main/LICENSE.txt',
-            serverLicenseUrl: 'https://github.com/microsoft/vscode/blob/main/LICENSE.txt'
+            licenseUrl: '',
+            serverLicenseUrl: ''
        });
    }
}
diff --git a/esm/vs/base/common/codiconsLibrary.js b/esm/vs/base/common/codiconsLibrary.js
index 6dc2df373f4e0a3a7a86b279fb2fce44c65b1d20..2a5d4c2fb0f12de3bd8dd34f3e0d4f1278e6da0a 100644
--- a/esm/vs/base/common/codiconsLibrary.js
+++ b/esm/vs/base/common/codiconsLibrary.js
@@ -97,9 +97,6 @@ export const codicons = new CodiconRegistry({
     folder: register('folder', 0xea83),
     fileDirectory: register('file-directory', 0xea83),
     symbolFolder: register('symbol-folder', 0xea83),
-    logoGithub: register('logo-github', 0xea84),
-    markGithub: register('mark-github', 0xea84),
-    github: register('github', 0xea84),
     terminal: register('terminal', 0xea85),
     console: register('console', 0xea85),
     repl: register('repl', 0xea85),
@@ -243,8 +240,6 @@ export const codicons = new CodiconRegistry({
     gitCompare: register('git-compare', 0xeafd),
     compareChanges: register('compare-changes', 0xeafd),
     gitMerge: register('git-merge', 0xeafe),
-    githubAction: register('github-action', 0xeaff),
-    githubAlt: register('github-alt', 0xeb00),
     globe: register('globe', 0xeb01),
     grabber: register('grabber', 0xeb02),
     graph: register('graph', 0xeb03),
@@ -414,7 +409,6 @@ export const codicons = new CodiconRegistry({
     runAll: register('run-all', 0xeb9e),
     syncIgnored: register('sync-ignored', 0xeb9f),
     pinned: register('pinned', 0xeba0),
-    githubInverted: register('github-inverted', 0xeba1),
     serverProcess: register('server-process', 0xeba2),
     serverEnvironment: register('server-environment', 0xeba3),
     pass: register('pass', 0xeba4),
@@ -565,7 +559,6 @@ export const codicons = new CodiconRegistry({
     runAllCoverage: register('run-all-coverage', 0xec2d),
     coverage: register('coverage', 0xec2e),
-    githubProject: register('github-project', 0xec2f),
     mapVertical: register('map-vertical', 0xec30),
     foldVertical: register('fold-vertical', 0xec30),
